target/linux/sunxi/image/cortex-a7.mk

```diff
@@ -47,6 +47,16 @@ define Device/friendlyarm_nanopi-neo-air
 endef
 TARGET_DEVICES += friendlyarm_nanopi-neo-air
 
+define Device/friendlyarm_nanopi-r1
+  DEVICE_VENDOR := FriendlyARM
+  DEVICE_MODEL := NanoPi R1
+  DEVICE_PACKAGES := kmod-rtc-sunxi kmod-usb2 kmod-usb-net-rtl8152 \
+ 	kmod-brcmfmac kmod-leds-gpio kmod-ledtrig-heartbeat wpad-basic \
+ 	brcmfmac-firmware-43430-sdio
```

Comment:
There seems to be a space before the tabs here. Just one tab please for the hanging indent.

Reply:
done